universe/Universe.cpp

```diff
@@ -2506,6 +2506,39 @@ void Universe::ApplyEffectDerivedVisibilities() {
     }
 }
 
+void Universe::ForgetKnownObject(int empire_id, int object_id) {
```

Comment:
ah... ok... local client needs to remove from the main objectmap, whereas on the server it applies to the empire-specific object knowledge...